### ISSUE TITLE
Atomic Sites: The "Try & Customize" option is not working for Varia-child themes

### ISSUE THREAD

**Issue by AtrumGeost** [wordpress-mobile/WordPress-iOS#16290]
### Expected behavior

For the theme preview to normal load

### Actual behavior

The mobile app (on iPhone and iPad) shows the following error:

`The parent theme is missing. Please install the "varia-wpcom" parent theme.`

![IMG_AC57BE7EDC33-1](https://user-images.githubusercontent.com/3696121/114579721-c06c5880-9c43-11eb-8021-9ddc8c90e8fd.jpeg)

### Steps to reproduce the behavior

Select an Atomic site, then:

1. Go to Themes
2. Tap on the "three-dots" next to a varia-child theme like Coutorie
3. Select "Try and Customize"

##### Tested on iPad Air 2 and iPhone Xs, iOS 14.4.2, WPiOS 17.0

**Comment by dangermattic** [comment]
Thanks for reporting! 👍
